app/components/task/holder.hbs

```diff
@@ -32,12 +32,25 @@
         >
           {{#each this.availabletaskStatusList as |taskStatus|}}
             {{#if (not-eq taskStatus.key this.TASK_KEYS.ALL)}}
-              <option
-                value={{taskStatus.key}}
-                selected={{eq taskStatus.key this.status}}
-              >
-                {{taskStatus.displayLabel}}
-              </option>
+              {{#if @dev}}
+                  {{#if (not (or (eq taskStatus.key this.TASK_KEYS.AVAILABLE) (eq taskStatus.key this.TASK_KEYS.COMPLETED)))}}
+                    <option
+                      value={{taskStatus.key}}
+                      selected={{eq taskStatus.key this.status}}
+                    >
+                      {{taskStatus.displayLabel}}
+                    </option>
+                {{/if}}
+              {{else}}
+                  {{#if (not (or (eq taskStatus.key this.TASK_KEYS.DONE)))}}
```

Comment:
If we add new logic under the old flow, won't it defeat the purpose of adding a feature flag?

Reply:
We have added new task status to constants file. So have to do this.
Can we do it in any other way?